src/tests/system/tests/test_passkey.py

```diff
@@ -592,3 +594,115 @@ def test_passkey__check_tgt(client: Client, ipa: IPA, moduledatadir: str, testda
 
     assert rc == 0, "Authentication failed"
     assert "Ticket cache" in output, "Failed to get the TGT"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.require.with_args(passkey_requires_root)
+def test_passkey__ipa_server_offline(
+    client: Client, ipa: IPA, moduledatadir: str, testdatadir: str, umockdev_ipaotpd_update
+):
+    """
+        :title: Check the authentication of user after kdestroy and when ipa service stop.
+    :setup:
+        1. Add a user with --user-auth-type=passkey in the server with passkey mapping.
+        2. Setup SSSD client with FIDO and umockdev, start SSSD service.
+    :steps:
+        1. Check authentication of the user and TGT after authentication.
+        2. Remove the tgt using #kdestroy -A and stop the IPA service.
+        3. Check the authentication again.
+        4. Check the feedback console message
+    :expectedresults:
+        1. User authenticates successfully and gets the TGT.
+        2. Successfully remove the TGT and IPA is not reachable.
+        3. User authenticate successfully, did not get TGT of user
```

Comment:
nitpick: some steps contain periods; some do not. Let's make it consistent.

Reply:
updated, please check again.